### DISCUSSION TITLE
Error when searching with Meilisearch

### DISCUSSION THREAD

**Discussion by pierpo** [AntlerVC/firetable#1379] [Support / Q&A]
Hey!

I activated the search option using Meilisearch. It works great on many collections, but I have an issue with my user collections.
It shows the following error: `Cannot read property of null (reading '_seconds')`.

Any idea?

The search works on Meilisearch's side (using their "preview search" feature). If I inspect the network on Rowy's side, I see that the user that I find is indeed returned by Meilisearch. I think the error comes from Rowy.

<img width="639" alt="image" src="https://github.com/rowyio/rowy/assets/6313316/7d9263c6-63e9-4cf1-9624-f20a2f0c758d">

**Comment by harinij** [comment]
Hi @pierpo is there any field with that keyname? Could you share your console logs or do a recording via https://jam.dev/ so we can investigate this further.

**Comment by harinij** [comment]
Closing older issue, in case the issue still persists you can reopen by commenting on it.

**Reply by pierpo** [reply]
Sorry I forgot to answer! My company went back to a base plan so I couldn't test again. Sorry about that :cry: 

Thanks for your support!
